src/app/components/error/error.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode } from 'react';
+
+import { Header } from '@app/components/header';
+import { useRouteHeader } from '@app/common/hooks/use-route-header';
+
+import { ErrorLayout } from './error.layout';
+
+interface ErrorProps {
+  body: string;
+  helpText: ReactNode[];
+  onClose?(): void;
+  title: string;
+}
+export function Error(props: ErrorProps) {
```

Comment:
I think `Error` alone is a bit too broad of a description, it also conflicts namespace with [`Error`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)

`<GenericErrorPage />` ? I think this works well because there's the relationship that a _SpecificErrorPage_ composes the _GenericErrorPage_.

Reply:
Yeah, agree here, will change. Are we wanting to add `Page` to all our page components. It seems like sometimes we have it but other times not.